### PR TITLE
DOCS-10053 Adding new optional env vars.

### PR DESCRIPTION
**Pull request by sleslie23**

### CODE REVIEW

**Review by kmendiratta-crm — DISMISSED** [review]

File: modules/ROOT/pages/install-aws.adoc [hunk 1]
```diff
@@ -122,7 +122,6 @@ Refer to xref:manage-index.adoc[Manage Runtime Fabric] for detailed information.
 *Required Environment Variables*
 
 [%header,cols="3*a"]
-.Required Environment Variables
```
Comment (kmendiratta-crm) [inline modules/ROOT/pages/install-aws.adoc:125]: we should still have the required variables section?

Comment (kmendiratta-crm) [reply modules/ROOT/pages/install-aws.adoc:125]: And perhaps just add disable_selinux to optional variables section? @sarjeet2013?

Comment (sleslie23) [reply modules/ROOT/pages/install-aws.adoc:125]: I'm removing the table titles, not the headings.

File: modules/ROOT/pages/install-aws.adoc [hunk 2]
```diff
@@ -138,18 +137,25 @@ Refer to xref:manage-index.adoc[Manage Runtime Fabric] for detailed information.
...
+| `existing_subnet_ids` | An list of existing subnet IDs within AWS. Required when installing on an existing network. The installer uses these subnets instead of creating a new one. |
+| `existing_vpc_id` | The VPC ID of an existing AWS Virtual Private Cloud (VPC). Required when installing on an existing VPC. The installer uses this VPC instead of creating a new one. |
+| `http_proxy` | The hostname and port for a HTTP proxy server to forward outbound HTTP requests. | `1.1.1.1:80`
+| `installer_url` | The URL of the Runtime Fabric installer package. | 
+| `kubernetes_api_cidr_blocks` | A cidr range that enables access to the kubernetes API. The default is an empty array: `[]`. | 
+| `monitoring_proxy` | The hostname and port, and optional username and password for a SOCKS5 proxy server to forward metrics and logs to Anypoint Monitoring. | `user:pass@1.1.1.2:800`
+| `mule_license` | The base64 content of your Mule license. | 
+| `no_proxy` | A comma-separated list of hosts which should bypass the proxy. | `1.1.1.1,no-proxy.com`
+| `ops_center_cidr_blocks` | A cidr range that allows access to the Ops Center UI. The default is an empty array: `[]`. |
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/install-aws.adoc]: CIDR not cidr "A CIDR range..."

File: modules/ROOT/pages/install-aws.adoc [hunk 3]
```diff
@@ -138,18 +137,25 @@ Refer to xref:manage-index.adoc[Manage Runtime Fabric] for detailed information.
 
 
 [%header,cols="3*a"]
-.Optional Environment Variables
 |===
 |Variable | Description | Example
+| `controllers` | The number of controller machines to provision. The default value is `3`. | `5`
+| `disable_selinux` | A boolean variable that enables/disables SELinux on the host operating system. The default is `false`. | `true`
+| `enable_public_ips` | A boolean varaible that indicates if public IP addresses should be specified. The default value is `false`. | `true`
+| `existing_subnet_ids` | An list of existing subnet IDs within AWS. Required when installing on an existing network. The installer uses these subnets instead of creating a new one. |
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/install-aws.adoc]: "A list" not "An list"

File: modules/ROOT/pages/install-aws.adoc (hunk 2 above)
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/install-aws.adoc]: "A CIDR range" not "A cidr range"

**Review by Mulesoft-Mysti — DISMISSED** [review]
wee typos

File: modules/ROOT/pages/install-aws.adoc (hunk 2 above)
Comment (sleslie23) [reply modules/ROOT/pages/install-aws.adoc]: Fixed

File: modules/ROOT/pages/install-aws.adoc (hunk 3 above)
Comment (sleslie23) [reply modules/ROOT/pages/install-aws.adoc]: Fixed

File: modules/ROOT/pages/install-aws.adoc (hunk 2 above)
Comment (sleslie23) [reply modules/ROOT/pages/install-aws.adoc]: Fixed

File: modules/ROOT/pages/install-aws.adoc [hunk 4]
```diff
@@ -138,18 +137,25 @@ Refer to xref:manage-index.adoc[Manage Runtime Fabric] for detailed information.
 
 
 [%header,cols="3*a"]
-.Optional Environment Variables
 |===
 |Variable | Description | Example
+| `controllers` | The number of controller machines to provision. The default value is `3`. | `5`
+| `disable_selinux` | A boolean variable that enables/disables SELinux on the host operating system. The default is `false`. | `true`
```
Comment (kmendiratta-crm) [inline modules/ROOT/pages/install-aws.adoc:143]: slight reword: "A boolean variable that disables SELinux.."
It comes enabled out of the box on certain operating systems, we only disable it when this var is passed.

**Review by kevintroller — APPROVED** [review]

**Pull request merged**